cargo-process.el

```diff
@@ -126,6 +130,42 @@
       (rename-buffer buffer))
     (set-process-sentinel (get-buffer-process buffer) 'cargo-process--finished-sentinel)))
 
+
+;; I/O tools
+
+
+(defun cargo-process--get-current-test ()
+  "Return the current test."
+  (let ((start (point))
+        name)
+    (save-excursion
+      (end-of-line)
+      (unless (and
+               (search-backward-regexp
+                (s-concat "^[[:space:]]*fn[[:space:]]*") nil t)
+               (save-excursion (go-end-of-defun) (< start (point))))
```

Comment:
A Mistake. It must be `rust-end-of-defun`. Defined in rust-mode [1]. 

[1] : https://github.com/rust-lang/rust-mode/blob/master/rust-mode.el